React/tic-tac/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=http://localhost:3001/api/
```

Comment:
As @WFolini said in a previous comment, this file usually contains critical data, please add it to the `.gitignore`.